gtda/mapper/pipeline.py

```diff
@@ -200,8 +200,8 @@ def make_mapper_pipeline(scaler=None,
         in a :obj:`joblib.parallel_backend` context. ``-1`` means using all
         processors.
 
-    parallel_backend_prefer : ``"processes"`` | ``"threads"``, optional, \
-        default: ``"threads"``
+    parallel_backend_prefer : ``"processes"`` | ``"threads"`` | ``None``, \
+        optional, default: ``"threads"``
```

Comment:
I agree! And I meant to change it (to `None` actually, but `None` is the same as processes and the joblib documentation is incorrect about this).

Reply:
Just pointing out that we are in mapper here. But your point is equally valid. Just to explain the history: the default `'threads'` was chosen here because it was observed to play better with the default `DBSCAN` clusterer for mapper.

Reply:
Oh yes, right, thats mapper. Sorry. :)